stripe/src/main/java/com/stripe/android/model/Stripe3ds2AuthResult.java

```diff
@@ -1,4 +1,635 @@
 package com.stripe.android.model;
 
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.stripe.android.utils.ObjectUtils;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static com.stripe.android.model.StripeJsonUtils.optString;
+
 public final class Stripe3ds2AuthResult {
+    private static final String FIELD_ID = "id";
+    private static final String FIELD_OBJECT = "object";
+    private static final String FIELD_ARES = "ares";
+    private static final String FIELD_CREATED = "created";
+    private static final String FIELD_ERROR = "error";
+    private static final String FIELD_LIVEMODE = "livemode";
+    private static final String FIELD_SOURCE = "source";
+    private static final String FIELD_STATE = "state";
+
+    @Nullable public final String id;
```

Comment:
let's make `id`, `object`, `created`, `livemode`, and `source` be annotated as `@NonNull`.